prometheus_client/openmetrics/parser.py

```diff
@@ -364,6 +342,32 @@ def _parse_nh_struct(text):
         pos_deltas=pos_deltas,
         neg_deltas=neg_deltas
     )
+  
+
+def _compose_spans(spans, spans_name):
+    try:
+        pos_spans_text = spans[spans_name]
+        pos_spans = []
+        for start, end in pos_spans_text:
```

Comment:
That _text is Hungarian notation :)
```suggestion
        for start, end in spans[spans_name]:
```